### ISSUE TITLE
Issue with keyboard echo in browsers when keyboard layout is switched

### ISSUE THREAD

**Issue by burakyuksek** [nvaccess/nvda#12373]
### Steps to reproduce:
Make sure you have more than 2 keyboard layouts.
1. Open google chrome
2. Enable keyboard echo
3. Type text into address bar or some edit box in a webpage
4. Hold alt, press shift twice to switch to third keyboard layout
5. Repeat step 3.
### Actual behavior:
You will not hear keyboard echo after step 5
### Expected behavior:
NVDA should still announce keyboard echo after step5
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.4
#### Windows version:
20H2
#### Name and version of other software in use when reproducing the issue:
Chrome 90.0.4430.93
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Previous versions behaved the same.
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
[nvda-old.log](https://github.com/nvaccess/nvda/files/6436432/nvda-old.log)

**Comment by amirsol81** [comment]
I can confirm it although the issue goes beyond browsers -- see issues #9816 #11058 and #11917 . Unfortunately despite lots of efforts to report this, no fix has been introduced so far. For the record, JAWS for Windows doesn't exhibit this issue at all.

**Comment by Adriani90** [comment]
This is still reproducible in NVDA 2024.4.1. Closing in favor of #9816 which is the same issue and contains more technical information.
